src/devices/BrickPi3/Extensions/EnumExtension.cs

```diff
@@ -15,7 +15,7 @@ public static T Next<T>(this T src)
         {
             if (!typeof(T).IsEnum)
             {
-                throw new ArgumentException($"Argumnent {typeof(T).FullName} is not an Enum");
+                throw new Exception($"Argument {typeof(T).FullName} is not an Enum");
```

Comment:
Why the switch from `ArgumentExcpetion` to plain `Exception`?

Reply:
Switched back (locally; will push before merging).